tests/test_utils.py

```diff
@@ -77,3 +78,16 @@ def test_update_dict():
 
     result = update_dict(first, third)
     assert "spaniel" not in result['animals']['dogs']
+
+
+def test_check_gender():
```

Comment:
feels like you are just testing python's random module here, not your code
also need a test to confirm WrongArgument is raised when appropriate